conf/quincy/rados/stretch-mode-host-location-attrs.yaml

```diff
@@ -59,5 +59,8 @@ globals:
         no-of-volumes: 4
         disk-size: 25
       node8:
+        image-name:
+          openstack: RHEL-9.2.0-x86_64-ga-latest
+          ibmc: rhel-91-server-released
```

Comment:
@SrinivasaBharath Please include these changes in #2669

Reply:
Included the changes.